test/images/apparmor-loader/BASEIMAGE

```diff
@@ -1,4 +1,4 @@
-amd64=alpine:3.8
-arm=arm32v6/alpine:3.8
-arm64=arm64v8/alpine:3.8
-ppc64le=ppc64le/alpine:3.8
+linux/amd64=alpine:3.8
+linux/arm=arm32v6/alpine:3.8
```

Comment:
I thought it was armhf?

Reply:
@PatrickLang we have been using this for a while now - https://cs.k8s.io/?q=arm32v6&i=nope&files=&repos=kubernetes/kubernetes